Harder better faster stronger

JohanObrink opened the pull request:
Fixes on infra
Custom command on tunnel
Updated README

raduachim commented on package.json:
```diff
@@ -24,15 +24,11 @@
     "localtunnel": "2.0.0",
     "subcommander": "1.0.0"
   },
-  "devDependencies": {
```
why? need those for semantic-release

JohanObrink commented:
Sorry! Merge b0rk :(

På 14 oktober 2019 till 10:02:18, Radu Achim (notifications@github.com)
skrev:

> *@raduachim* commented on this pull request.
> ------------------------------
>
> In package.json
> <https://github.com/egendata/cli/pull/2#discussion_r334362046>:
>
> >      "ip": "1.1.5",
>      "localtunnel": "2.0.0",
>      "subcommander": "1.0.0"
>    },
> -  "devDependencies": {
>
> why? need those for semantic-release
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/egendata/cli/pull/2?email_source=notifications&email_token=AAEF7L3ZPG6V4OR2TIYIPR3QOQRQVA5CNFSM4I7ZNSXKYY3PNVWWK3TUL52HS4DFWFIHK3DMKJSXC5LFON2FEZLWNFSXPKTDN5WW2ZLOORPWSZGOCHZS57A#pullrequestreview-301149948>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AAEF7L7FFC7ODCQSV6GV7XLQOQRQVANCNFSM4I7ZNSXA>
> .
>

JohanObrink commented:
It didn’t work when I added more yml files. It’s a feature that should be
added.

På 14 oktober 2019 till 10:22:11, Radu Achim (notifications@github.com)
skrev:

> *@raduachim* requested changes on this pull request.
>
> One thing that we seemed to have lost, although I couldn't get it to work
> on older revision of this package is the option to pull latest images --pull
> – Forces a pull of the latest Egendata images
>
> I think the pull should sort of be done by default on the operator image.
> Currently I am running an older version of it and it took me a while to
> figure out why the operator keeps crashing with the new configuration from
> the yaml file
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/egendata/cli/pull/2?email_source=notifications&email_token=AAEF7L5HGW2RLE6OFQW347TQOQT3HA5CNFSM4I7ZNSXKYY3PNVWWK3TUL52HS4DFWFIHK3DMKJSXC5LFON2FEZLWNFSXPKTDN5WW2ZLOORPWSZGOCHZU6YA#pullrequestreview-301158240>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AAEF7L7R3TO5E3DF7ULFICLQOQT3HANCNFSM4I7ZNSXA>
> .
>

raduachim commented on package-lock.json:
```diff
@@ -4,6 +4,8 @@
   "dependencies": {
+<<<<<<< HEAD
+=======
     "@babel/code-frame": {
```
we should recreate this package-lock. it's b0rked over 9000

egendata-dev commented:
:tada: This PR is included in version 0.2.0 :tada:

The release is available on:
- [npm package (@latest dist-tag)](https://www.npmjs.com/package/@egendata/cli/v/0.2.0)
- [GitHub release](https://github.com/egendata/cli/releases/tag/v0.2.0)

Your **[semantic-release](https://github.com/semantic-release/semantic-release)** bot :package::rocket:
